src/devices/Pn5180/Pn5180.cs

```diff
@@ -441,6 +442,31 @@ public override int Transceive(byte targetNumber, ReadOnlySpan<byte> dataToSend,
                         return TransceiveBuffer(dataToSend, dataFromCard);
                 }
             }
+            else if (protocol == NfcProtocol.Iso15693)
+            {
+                switch ((IcodeCardCommand)dataToSend[1])
+                {
+                    case IcodeCardCommand.ReadSingleBlock:
+                    case IcodeCardCommand.WriteSingleBlock:
+                    case IcodeCardCommand.LockBlock:
+                    case IcodeCardCommand.ReadMultipleBlocks:
+                    case IcodeCardCommand.WriteMultipleBlocks:
+                    case IcodeCardCommand.StayQuiet:
+                    case IcodeCardCommand.Select:
+                    case IcodeCardCommand.ResettoRead:
+                    case IcodeCardCommand.LockAFI:
+                    case IcodeCardCommand.LockDSFID:
+                    case IcodeCardCommand.GetSystemInformation:
+                        var ret = SendDataToCard(dataToSend.ToArray());
+                        if (!ret)
+                        {
+                            return -1;
+                        }
+
+                        Thread.Sleep(2 + dataToSend.Length / 3);
+                        return ReadWithTimeout(dataFromCard, 2 + dataFromCard.Length / 3);
+                }
```

Comment:
in the other case, you may want to throw an exception maybe? Or can anything else be part of this protocol?